spec/integration/elastic_integration_spec.rb

```diff
@@ -1134,6 +1134,27 @@ def path; @path; end
       # end
     end
 
+    describe 'with terminate processor' do
+      let(:pipeline_processor) {
+        '{
+          "terminate": {
+            "if": "ctx.error != null",
+            "tag": "terminated_ingest_pipeline"
```

Comment:
I wished this tag be available in `_ingest_document` so that I can `expect(event.get("[tag]")).to eql? "terminated_ingest_pipeline"` but no tag exists.